.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -190,8 +190,8 @@ stages:
           scriptType: pscore
           scriptLocation: inlineScript
           inlineScript: |
-            
-            Do {
+
+            do {
```

Comment:
I still have an itch to rewrite all these scripts to "proper" PowerShell, but then I realize how many we have.... :D So this is ok I guess.